lib/ansible/cli/playbook.py

```diff
@@ -136,6 +137,9 @@ def run(self):
 
         # create base objects
         loader, inventory, variable_manager = self._play_prereqs()
+        env_vars = load_env_vars(loader)
+        if env_vars:
+            variable_manager.extra_vars.setdefault('ansible_env', {}).update(env_vars)
```

Comment:
This part is incorrect, `ansible_env` is a reflection of gathered facts, not of variables that Ansible itself has been invoked with.